src/Widgets/TopNavigationWidget/TopNavigationWidgetEditingConfig.ts

```diff
@@ -4,6 +4,10 @@ import Thumbnail from './thumbnail.svg'
 
 provideEditingConfig(TopNavigationWidget, {
   attributes: {
+    metaNavigationUtilityLink: {
+      title: 'Utility link',
+      description: 'Shown next to the current user dropdown.',
```

Comment:
* [ ] FYI the "current user" item may be hidden. How about "next to the language switch"?

Reply:
The language switch may be hidden as well 😸

Reply:
> The language switch may be hidden as well 😸

True, but not by default.
Maybe just drop the description as this can be explored in WYSIWYG properties editing.